### ISSUE TITLE
DataTable: expanded row styling

### ISSUE THREAD

**Issue by ribizli** [primefaces/primeng#1845]
**I'm submitting a ...**
```
[ ] bug report => Search github for a similar issue or PR before submitting
[x] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
Expanded row has no class on it, no way to style it differently (e.g. remove padding}

**Expected behavior**
Make it possible to differentiate the expanded row from the others.

**Minimal reproduction of the problem with instructions**

**What is the motivation / use case for changing the behavior?**
We need to change the padding in the expanded row easily.
Workaround is to use `p-datatable tbody > tr:not(.ui-widget-content)` pseudo selector, but it might be not restrictive enough.

**Please tell us about your environment:**

* **Angular version:** latest

* **PrimeNG version:** 1.1.4

* **Browser:** [all]
 
* **Language:** [all]

**Comment by ricardasjak** [comment]
So the Fix #1845 was not merged, this issue still is relevant, I propose to re-open this ticket. @jsmucr @cagataycivici

**Comment by ribizli** [comment]
Use TurboTable, you have control there...
